appveyor.yml

```diff
@@ -6,16 +6,13 @@ install:
   # Get the latest stable version of Node.js or io.js
   - ps: Install-Product node $env:nodejs_version
   - ps: $env:BuildNumber= $env:APPVEYOR_BUILD_NUMBER
+  - ps: $env:Configuration="Release"
 init:
   - git config --global core.autocrlf true
 build_script:
-  - dotnet restore
```

Comment:
The Visual Studio 2017 image for AppVeyor CI. In Travis CI it's more difficult. Maybe KoreBuild is too much for us. Maybe we need to use dotnet-install?